src/signals/incident-management/components/FilterForm/__tests__/actions.test.ts

```diff
@@ -69,7 +70,8 @@ describe('signals/incident-management/components/FilterForm/actions', () => {
 
   it('should create an action to set categories', () => {
     const slug = 'foo-bar'
-    const subCategories = structuredCategories['openbaar-groen-en-water'].sub
+    const subCategories = structuredCategories['openbaar-groen-en-water']
+      .sub as unknown as ExtendedSubCategory[]
```

Comment:
what's happening here?

Reply:
as unknown as Ex...

Reply:
I'm casting it to ExtendedSubCategories, but since it misses some properties I need to cast is to unknown first. Honestly, the types around this Category, ExtendedCategory and StructuredCategory should be revisited. But it's out of scope for this PR.

Reply:
coolio